healsparse/healSparseMap.py

```diff
@@ -164,7 +164,7 @@ def read(cls, filename, nside_coverage=None, pixels=None, header=False,
                     row = fits.read_ext_data(1, row_range=[0, 1])
                     dtype = row[0][0][0].dtype.type
 
-            healpix_map = hp.read_map(filename, nest=True, verbose=False, dtype=dtype)
+            healpix_map = hp.read_map(filename, nest=True, dtype=dtype)
```

Comment:
Was this change intentional?

Reply:
Yes, this was a separate commit.  I removed it because the `verbose` keyword has been deprecated and does nothing.